UniMath/wc_count_monoidal.l

```diff
@@ -0,0 +1 @@
+../../monoidal/wc_count_monoidal.sh
```

Comment:
What's this about?

Reply:
This must have inadvertently been committed. The shell script in question quantifies recent contributions made to UniMath that are described in an upcoming workshop talk.